lib/theme.dart

```diff
@@ -141,15 +141,17 @@ class AppTheme {
     );
   }
 
-  static ThemeData getThemeFromPreferences(bool automatic, bool dark) {
-    DateTime now = DateTime.now();
-    if (automatic) {
-      if (now.hour >= 20 || now.hour < 8) return AppTheme.darkTheme();
-      else return AppTheme.lightTheme();
-    } else {
+  static ThemeData getNormalThemeFromPreferences(bool automatic, bool dark) {
```

Comment:
Why `getNormalXXX` ? Where is the `getAbnormalXXX` ?
```suggestion
  static ThemeData getThemeFromPreferences(bool isAutomatic, bool isDark) {
```